.travis.yml

```diff
@@ -7,6 +7,7 @@ env:
         - TOX_ENV=py27-django19
         - TOX_ENV=py27-django110
         - TOX_ENV=py27-django111
+        - TOX_ENV=flake8
```

Comment:
This seems like the right PR to modify the env name to be `py27-flake8`. It will get updated to py37-flake8 of course, but doing this now will make that easier to remember.

Reply:
This would need to be changed in tox.ini as well